### ISSUE TITLE
"Server Configuration Manager" -  test the SSH connection and verify access to URLs (Mapbender and QGIS server)

### ISSUE THREAD

**Issue by pfavero-wheregroup** [WhereGroup/QGIS2Mapbender#4]
In the "Server Configuration Manager" tab, it is useful to be able to test the SSH connection and verify access to the Mapbender and QGIS server URLs.

the following steps could be  performed:
            1. Establishes an SSH connection using the provided SSH client.
            2. Attempts to access a specified QGIS Project path.
            3. Attempts to access a specified Mapbender's path.
            4. Check the validity of the Mapbender URL and verify that the response is 200.
            5. Check the validity of the QGIS URL and verify that the response is 200.

**Comment by astroidex** [comment]
Hello @pfavero-wheregroup sounds like a good feature. How much time do you think would be needed to implement it.

**Comment by pfavero-wheregroup** [comment]
Yes, this is a new feature. It can  be very helpful to avoid saving a wrong configuration in QGIS, which will be difficult to find it later when the uploading of a QGIS project to Mapbender is performed. This happened last week when @kartoffelfee and I installed everything on a new server (QGIS Server, Mabender, etc) and it was hard to find out what was wrong in the configuration....

Time needed: about 4/5 hours
